src/components/SettingsModal.js

```diff
@@ -12,6 +20,36 @@ export default function SettingsModal({ setDisplayModal }) {
           className="modal-close-button"
         />
         <h1 className="modal-header">Settings</h1>
+
+        <div className="modal-range-container">
+          <label htmlFor="delay-slider">
+            Set how frequently the app will fetch new places. Values are in
+            seconds.
+          </label>
+          <div className="modal-range-pic-container">
+            <span className="modal-range-pic">
+              <img src="./img/slider-car.png" alt="car-emoji" />
```

Comment:
Also totally non-blocking, but the `alt` text here could be improved by thinking of what information you're trying to convey to screen reader users. E.g., instead of "car-emoji", consider "No delay is best when traveling by car".

Reply:
Great call. 👍